src/kms/TenantSecurityClient.ts

```diff
@@ -192,6 +194,15 @@ export class TenantSecurityClient {
             .map((rekeyResponse) => ({encryptedDocument: encryptedDoc.encryptedDocument, edek: rekeyResponse.edek}))
             .toPromise();
 
+    /**
+     * Re-key a document's encrypted document key (EDEK) to a new tenant ID. Unwraps the EDEK via the original tenant's KMS, then re-wraps that DEK
```

Comment:
"document key" should be "document encryption key"

Reply:
As mentioned in other PR, this is the wording we use across all TSCs, so not going to change them all for now